doc/Changelog.md

```diff
@@ -1,29 +1,36 @@
 Changelog {#Changelog}
 =========
 
-# master {#master}
+# Release 0.7 (09-12-2016)
+* [#353](https://github.com/BlueBrain/Livre/pull/353):
+  handle files with right extension and no schema
 * [#352](https://github.com/BlueBrain/Livre/pull/352):
-  Fix missing redraw after expose
-* [#331](https://github.com/BlueBrain/Livre/pull/331):
-  Multiple pass mode added for high quality images for data that does not fit into GPU cache
+  Fix missing redraw after expose.
+  * Fix the %300 CPU usage.
+* [#346](https://github.com/BlueBrain/Livre/pull/346):
+  Implement axis and legend for full volume bounding box.
 * [#333](https://github.com/BlueBrain/Livre/pull/333):
-  Add animation-fps parameter to limit the animation frame rate
+  Add animation-fps parameter to limit the animation frame rate.
+* [#331](https://github.com/BlueBrain/Livre/pull/331):
+  Multiple pass mode added for high quality images for data that does not fit into GPU cache.
```

Comment:
* Added support for rendering high-resolution volumes at full resolution

(the rest are implementation details)